### PR TITLE
txn: Remove Command and rename CommandKind to Command

### PR DESCRIPTION
**Pull request by longfangsong**
Signed-off-by: longfangsong <longfangsong@icloud.com>

### What problem does this PR solve?

The only field in `Command` struct is `kind: CommandKind`, so we can use `CommandKind`(should be renamed into `Command`) directly.

### What is changed and how it works?

Rename the old CommandKind into Command and use it to replace the old Command struct, which only field is the CommandKind.

### Check List 

Tests 

- Unit test

### CODE REVIEW

**Comment by longfangsong** [comment]
@nrc PTAL.

File: src/storage/mod.rs [hunk 1]
```diff
@@ -33,15 +33,13 @@ pub use self::{
 
 use crate::read_pool::{ReadPool, ReadPoolHandle};
 use crate::storage::metrics::CommandKind;
+use crate::storage::txn::Command;
```
Comment (nrc) [inline src/storage/mod.rs:36]: Better to leave this import where it was.

**Comment by nrc** [comment]
@youjiali1995 PTAL

**Pull request merged**